doc/api/http.md

```diff
@@ -623,6 +624,31 @@ If `data` is specified, it is equivalent to calling
 If `callback` is specified, it will be called when the request stream
 is finished.
 
+### `request.destroy([error])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `error` {Error} Optional, an error to emit with `'error'` event.
+* Returns: {this}
+
+Destroy the request. Optionally emit an `'error'` event,
+and emit a `'close'` event. Calling this will cause remaining data
+in the response to be dropped and the socket to be destroyed.
+
+See [`writable.destroy()`][] for further details.
+
+#### `request.destroyed`
+<!-- YAML
+added: REPLACEME
```

Comment:
I think this was added in a previous release, right?

Reply:
https://github.com/nodejs/node/pull/32148

Included in the pending 14.x

Reply:
@mcollina: Was there anything you wanted to change here or can this land?

Reply:
This is ok then.